synthclient/manage-exemptions/README.md

```diff
@@ -0,0 +1,55 @@
+# manage-exemptions
+
+This tool is for SecureDNA users (e.g. DNA providers) who want to manage their customers' exemptions.
+
+## Background
+
+Ordinarily, when placing an order that has an exemption, customers submit a six-digit 2FA code that gets forwarded to synthclient and checked by SecureDNA. However, a provider may with to manage [exemption tokens](https://github.com/SecureDNA/SecureDNA/wiki/Requesting-exemptions) instead of handing them out to its customers. In this case, the provider becomes responsible not only for requesting and approving exemptions on behalf of its customers, but also for keeping the 2FA secrets and generating six-digit codes.
```

Comment:
```suggestion
Ordinarily, when placing an order that has an exemption, customers submit their exemption are also prompted for a six-digit 2FA code that gets forwarded to synthclient and checked by SecureDNA. However, a provider may wish to be its own biosafety authority, and to manage [exemption tokens](https://github.com/SecureDNA/SecureDNA/wiki/Requesting-exemptions) itself, instead of customers using an external BSA and managing their exemptions themselves. In this case, the provider becomes responsible not only for requesting and approving exemptions on behalf of its customers, but also for keeping the 2FA secrets and generating six-digit codes.
```